XamlControlsGallery/ControlPages/PullToRefreshPage.xaml.cs

```diff
@@ -88,7 +88,7 @@ public PullToRefreshPage()
                 {
                     Width = 200,
                     Height = 200,
-                    BorderThickness = new Thickness(1),
+                    BorderThickness = new Thickness() { Left = 1, Top = 1, Right = 1, Bottom = 1 },
```

Comment:
Is specifying every member needed now?

Reply:
This PR is a sync of changes that were made internally on WinUI3 with the purpose of unforking and removing the internal copy. If changes need to be made in either WinUI3 or in WinUI2, please file an issue and we will fix that after the sync is complete.  Thanks!